### ISSUE TITLE
places subform redesign

### ISSUE THREAD

**Issue by tuurma** [eXistSolutions/LGPN#60]
New model for places:

```xml
<place type="settlement" xml:id="{$id}" xmlns="http://www.tei-c.org/ns/1.0">
      <placeName type="" subtype="" cert="" xml:lang=""/>
...
      <placeName type="" subtype="" cert="" xml:lang=""/>
      <location type="pleiades">
              <label/>
       </location>
       <location cert="">
             <geo/>
             <alt/>
        </location>
</place>
```
@type: ancient | modern | other
@subtype: major | variant
@cert: high | low
@xml:lang: from the `languages.xml` list

Only name variants tagged as 'major' end up displayed by default (joined by hyphens) e.g.

`Palmyra-Tadmur`

**Comment by tuurma** [comment]
@michaelzellmann I will convert existing place records to this structure and upload changes tonight

![image](https://cloud.githubusercontent.com/assets/449468/25955238/95dacf4a-3660-11e7-83fb-ce5340b318c0.png)

**Comment by tuurma** [comment]
add place/@type: region | subregion | settlement

**Comment by tuurma** [comment]
current values for place/@type 

```xml
<data>
    <option value="settlement">settlement</option>
    <option value="region">region</option>
    <option value="island">island</option>
    <option value="deme">deme</option>
</data>
```

**Comment by tuurma** [comment]
![image](https://cloud.githubusercontent.com/assets/449468/25956966/1ad0f52c-3665-11e7-8be8-0965ba6e92dd.png)

**Comment by tuurma** [comment]
```xml

(: add type where none :)
for $i in doc('/db/apps/lgpn-data/data/volume0.places.xml')//tei:place/tei:placeName[not(@type)]
return
update insert attribute type {''} into $i

(: add subtype:)
for $i in doc('/db/apps/lgpn-data/data/volume0.places.xml')//tei:place/tei:placeName[not(@subtype)]
let $type:=$i/@type
let $subtype:=if($type ne '') then 'variant' else ''
return
update insert attribute subtype {$subtype} into $i

for $i in doc('/db/apps/lgpn-data/data/volume0.places.xml')//tei:place/tei:placeName[not(@cert)]
let $cert:=''
return
update insert attribute cert {$cert} into $i

for $i in doc('/db/apps/lgpn-data/data/volume0.places.xml')//tei:place/tei:placeName[not(@xml:lang)]
let $lang:=''
return
update insert attribute xml:lang {$lang} into $i
```

**Comment by tuurma** [comment]
closed with 9008c17e234f37901b4adcf9f0ab482a48fab23b

**Comment by tuurma** [comment]
Since information about place type is now explicitly stored in place/@type, wrappers like `<settlement>`, '<district>` etc are unnecessary and cause trouble when editing placeNames

```xquery
for $i in collection('/db/apps/lgpn-data/data/places')//tei:placeName[child::*]
    let $replacement := 
    element {node-name($i)} {
        for $att in $i/@* 
        return attribute {name($att)} {$att}
        ,
        normalize-space($i/string())
    }
return 
(:    $replacement:)
update replace  $i with $replacement
```
